main.cpp

```diff
@@ -346,6 +243,34 @@ Thing 2) Field Recorder
     2) set 192 khz/32 bit-float
     3)insert headphone
  */
+struct filedRecorder
+{
+  unsigned int inputs;
+  unsigned int outputs;
+  float sizeOfStorage;
+  double supportedFormats;
+  unsigned int settings;
+
+  struct neumannKm184
+  {
+    neumannKm184();
+    bool isThisPair = true;
+    int modelYear = 2018;
+    void frequencyRange(float lowFreq = 20.f, float highFreq = 20.000);
+    double pricing = 829.20;
+    bool isThisNew = false;
+
+    void openBox();
+    void selectCable(int numberOfCable = 2, std::string cableBrand = "Neutrik"); 
+    void insertCable(std::string cableType = "XLR");
+  };
+
+  void selectRecordingFormat(float sampleRate = 192.f, int bitRate = 32);
+  void startRecording();
+  void ejectMics(neumannKm184 mono1, neumannKm184 mono2);
+
+  neumannKm184 ejected; 
+};
```

Comment:
3 things need revising here:
- You have 4 variable declarations and 4 function declarations.  The instructions specified that 5 variables and 3 functions need to be declared.
- The code inside the `{ }` needs to be indented 4 spaces relative to the `{`
- Type names need to begin with a capitalized first letter. `FieldRecorder` and `NeumannKm184` is what you want to write. 
Please correct the issues
- [ ] revise